include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort_one_wg.h

```diff
@@ -169,7 +169,7 @@ struct __subgroup_radix_sort
                         _KeyT __keys[__block_size];
                         uint16_t __wi = __it.get_local_linear_id();
                         uint16_t __begin_bit = 0;
-                        constexpr uint16_t __end_bit = sizeof(_KeyT) * 8;
+                        constexpr uint16_t __end_bit = sizeof(_KeyT) * ::std::numeric_limits<unsigned char>::digits;
```

Comment:
@akukanov, could you please explain why you are using here ```::std::numeric_limits<unsigned char>::digits``` instead of  ```CHAR_BIT``` like in other places?
As described at https://en.cppreference.com/w/cpp/types/numeric_limits/digits you may use ```CHAR_BIT``` in this place too: 
```
Standard specializations
unsigned char     CHAR_BIT
```

Reply:
In this file, we already have the `<limits>` header included, so can use `numeric_limits`. In the other file, `<climits>` and `CHAR_BIT` seem preferable.

Reply:
OK